### PR TITLE
(cu 861m934yu) update audits content in repo

### PR DESCRIPTION
**Pull request by Martin-Kiss**
Contains both tasks CU-862j4nan0 and CU-862j4nan0

### CODE REVIEW

**Comment by KaiserKarel** [comment]
Task linked: [CU-862j4nan0 Add Halborn final report to github for Grandpa light client](https://app.clickup.com/t/862j4nan0)

**Comment by KaiserKarel** [comment]
Task linked: [CU-861m934yu Update audits content in repo](https://app.clickup.com/t/861m934yu)

File: audits/README.md [hunk 1]
```diff
@@ -5,48 +5,24 @@ The related code audits are provided for context.
...
-   Pallet:
-   - IBC (Inter-Blockchain-Communication)
-
-7. [Halborn Audit - 10.10.2022](./halborn/audit08-pallets-stakingRewards-fnft.pdf)
-
-   Pallets:
-   - Staking Rewards
-   - fNFT
-   
 ## Post-Mortems
```
Comment (0xBrainjar2) [inline audits/README.md:27]: no need to put this here, and again, keep everything that has been audited in here

**Review by KaiserKarel — APPROVED** [review]

**Review by PoisonPhang — APPROVED** [review]

**Review by 0xBrainjar2 — APPROVED** [review]

**Comment by Martin-Kiss** [comment]
@mergifyio update

**Comment by Martin-Kiss** [comment]
@mergifyio refresh

**Pull request merged**